modules/ivc_champva/app/controllers/ivc_champva/v1/uploads_controller.rb

```diff
@@ -41,28 +44,46 @@ def submit_supporting_documents
 
       private
 
-      def get_file_paths_and_metadata(parsed_form_data)
+      def get_attachment_ids_and_form(parsed_form_data)
         form_id = get_form_id
         form = "IvcChampva::#{form_id.titleize.gsub(' ', '')}".constantize.new(parsed_form_data)
-        filler = IvcChampva::PdfFiller.new(form_number: form_id, form:)
+        attachment_ids = [form_id]
+
+        if form_id == 'vha_10_10d'
```

Comment:
This is only going to work for 10-10D? Shouldn't we write this to work for any form?

Reply:
If your form wants to use this logic, you will have to modify the data structure for your form

Reply:
You have to change the data structure of your form to :  `"applicants"=>  and "applicant_supporting_documents"=>` if you want to apply this method.